src/element/newElement.ts

```diff
@@ -46,6 +46,7 @@ import {
 } from "../constants";
 import { isArrowElement } from "./typeChecks";
 import { MarkOptional, Merge, Mutable } from "../utility-types";
+export const SOURCE = Math.random().toString();
```

Comment:
This approach sets a different value for `SOURCE` every time the page reloads.  This could inflate the apparent number of people connected.